modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -269,6 +269,15 @@ bond=bond0:em1,em2:mode=active-backup
 vlan=bond0.100:bond0
 ----
 
+[discrete]
+=== Bonding multiple network interfaces to a single interface with mtu 
+
```

Comment:
There needs to be some descriptive text here, to keep it consistent with the other examples in this section. What do you think about the following:
```suggestion
Optional: You can configure an MTU on a VLAN subinterface by adding `bond0:none:9000` to the `ip=` parameter to specify the MTU, for example:
```

Reply:
Done.

Reply:
@Rupesh-git-eng I don't see the description update in the commit history, did you push the changes?

Reply:
Sorry, looks like it got lost during the update. Can you check it once again, please?

https://github.com/Rupesh-git-eng/openshift-docs/edit/patch-8/modules/installation-user-infra-machines-static-network.adoc?pr=%2Fopenshift%2Fopenshift-docs%2Fpull%2F58631